cmd/soroban-rpc/internal/methods/simulate_transaction.go

```diff
@@ -21,19 +21,21 @@ type SimulateTransactionCost struct {
 	MemoryBytes     uint64 `json:"memBytes,string"`
 }
 
-type SimulateTransactionResult struct {
-	Auth      []string `json:"auth"`
-	Events    []string `json:"events"`
-	Footprint string   `json:"footprint"`
-	// TODO: update documentation and review field name
-	XDRs []string `json:"xdrs"`
+// SimulateHostFunctionResult contains the simulation result of each HostFunction within each InvokeHostFunctionOp
+type SimulateHostFunctionResult struct {
```

Comment:
It seems that we have lots of "Simulate" prefix here - which is correct, but might not place the focus where it should be. How do you feel about the following naming for this struct:
```golang
type ExecutedFunction {
	Auth []string `json:"auth"`
	Results  string   `json:"results"` // base64 xdr encoded...
}
```
The field name that I feel is really the issue is the `XDR`, since this is a type rather then a data element descriptor.

Reply:
The XDR name predates this release iteration. Also, `results` is redundant with the higher level name, plus it's a single result of a single function, so either way I think it should be singular.

Reply:
I think that you agree with me that's it's undesirable, but don't have any concrete solution here. I'm not very opinionated here, since there is nothing structurally wrong, but want to try and figure out a better "names" for the constructs that we're creating.